app/controllers.js

```diff
@@ -0,0 +1,181 @@
+// Controllers
+
+// Home controller
+// Use the array to avoid the minification problem
+airTrafficControlApp.controller('homeController', ['$scope', '$location', 'systemBootService' , 'queueManagementService', function($scope, $location, systemBootService, queueManagementService){
+    
+    // If the system is already booted, redirect to
+    // the air traffic control page
+    if (true == systemBootService.systemIsBooted()){
+        $location.path("/airTrafficControlAndStatus");
+    }
+    
+    $scope.bootSystem = function(){
+
+        // Make async call to boot the system
+        systemBootService.bootSystem()
+        .then(function(bootStatus){
+            // redirect to the air traffic control view
+            queueManagementService.initialize();
+            $location.path("/airTrafficControlAndStatus");
+        })
+        .catch(function(errorMessage){
+            // If the deferred object were to reject,
+            // show an error message
+            alert("An error occured. " + errorMessage); 
+        });
+    };
+    
+}]);
+
+
+// System Status Controller
+// Use the array to avoid the minification problem
+airTrafficControlApp.controller('airTrafficControlAndStatus', ['$scope', '$timeout', 'systemBootService', 'queueManagementService', function($scope, $timeout, systemBootService, queueManagementService) {
+    
+    var self = this;
+    $scope.aircraftQueue = [];
+    enqueueTimeoutPromise = null;
+    dequeueTimeoutPromise = null;
+    
+    // If the system is not booted, redirect to
+    // home page
+    if (false == systemBootService.systemIsBooted()){
+        systemBootService.handleUnbootedSystemAccess();
+        return;
+    }
+    
+    // Aircraft specifications for the user
+    // selected value will be used as
+    // two way binding for user selection
+    $scope.aircraftSpecifications = {
+        "aircraftTypeSpecification":{
+        "title": "Aircraft Type",
+        "selectedValue": "Passenger"
+        },
+        "aircraftSizeSpecification":{
+        "title": "Aircraft Size",
+        "selectedValue": "Large"
+        }
+    };
+    
+    // As the queue length changes, we need to
+    // update the display. This display is
+    // for the list for of all aircrafts in the system
+    $scope.$watch(function(){

```

Comment:
can't this be accomplished by binding the correct object collection and then just having an ng-repeat? having watch instead of leverage native two way binding seems like extra work.